actionpack/lib/action_dispatch/http/permissions_policy.rb

```diff
@@ -3,6 +3,22 @@
 require "active_support/core_ext/object/deep_dup"
 
 module ActionDispatch # :nodoc:
+  # Configures the HTTP
+  # {Feature-Policy}[https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy]
+  # response header to allow or block the use of browser features in its own
+  # context or in iframes that it embeds.
```

Comment:
"in its own context" feels a little nebulous here.  Since all of the directives on the MDN page mention "current document", what about using that?

```suggestion
  # response header to allow or deny the current document and its iframes the 
  # use of specific browser features.
```

Or:

```suggestion
  # response header to specify which browser features the current document and 
  # its iframes can use.
```

Reply:
Yes, I think current document is a lot clearer.